lib/Services/ExtractMetadata.php

```diff
@@ -44,11 +44,15 @@ public function extract($absoluteImagePath) {
 	 * @return array
 	 */
 	private function extractImageDimensions($absoluteImagePath) {
-		$dimensions = getimagesize($absoluteImagePath);
-
 		$logger = \OC::$server->getLogger();
 		$logger->debug('Image Path: {absolutePath}', array('app' => 'MediaMetadata', 'absolutePath' => $absoluteImagePath));
 
+		try {
+			$dimensions = getimagesize($absoluteImagePath);
+		} catch (\PHPUnit_Framework_Exception $e) {
```

Comment:
This is not a test, don't use PHP Unit exceptions here. Create your own Service exceptions later. For now, just use \Exception.